charts/postgres-operator/values.yaml

```diff
@@ -462,7 +462,10 @@ podServiceAccount:
 priorityClassName: ""
 
 # priority class for database pods
-podPriorityClassName: ""
+podPriorityClassName:
+  create: true
```

Comment:
Lets copy the behavior of the service account. If create is true a priority class should get created and if name is empty then we use the default `default (printf "%s-%v" (include "postgres-operator.fullname" .) "pod")` (see _helpers.tpl file)